match2/wikis/starcraft2/match_group_legacy.lua

```diff
@@ -132,10 +129,29 @@ function Legacy._convert(mapping)
 			end
 
 			if not Logic.isEmpty(nested) then
+				local score1 = json.parseIfString(nested.opponent1 or {}).score or ''
+				local score2 = json.parseIfString(nested.opponent2 or {}).score or ''
+
+				--handle advantages that were passed the old way
+				local score1adv, score1sum = string.match(score1,
+					'<abbr title="Winner\'s bracket advantage of (%d) game">(%d)</abbr>')
+				if score1adv then
+					nested.opponent1 = json.parseIfString(nested.opponent1 or {})
+					nested.opponent1.score = score1sum
+					nested.opponent1.advantage = score1adv
+					nested.opponent1 = json.stringify(nested.opponent1)
```

Comment:
Since this code is duplicated, use a function? That allows you to get rid of all the `score1sum`, etc too

Reply:
done